summit/2018.html

```diff
@@ -41,16 +41,16 @@ <h2 class="location">Location</h2>
         <div class="col-md-10 col-md-offset-1">
           <img class="schedule" src="/images/summit/schedule.svg" width="65px" height="auto" alt="Schedule"/>
           <h2 class="text">Schedule</h2>
-          <p><strong>2 November</strong><br/>
-          4 PM - 8 PM: Reception and Planning for the Week-end Topics</p>
-          <p><strong>3 November</strong><br/>
+          <p class="schedule-text"><strong>2 November<br/>
+          4 PM - 8 PM: Reception and Planning for the Week-end Topics</strong></p>
+          <p class="schedule-text"><strong>3 November<br/>
           10 AM - 1 PM: Open Source Design Meta Discussions<br />
           1 PM - 2 PM: Lunch<br />
-          2 PM - 8 PM: Unconference Sessions</p>
-          <p><strong>4 November</strong><br />
+          2 PM - 8 PM: Unconference Sessions</strong></p>
+          <p class="schedule-text"><strong>4 November<br />
           10 AM - 1 PM: Working Sessions Hands-on<br />
           1 PM - 2 PM: Lunch<br />
-          2 PM - 6 PM: Working Sessions & Open Source Design in 2019</p>
+          2 PM - 6 PM: Working Sessions & Open Source Design in 2019</strong></p>
```

Comment:
I preferred the old version, without the strong surrounding the whole schedule. It was better for scanning of the days.

Reply:
Kinda same to be honest

Reply:
But I can make the days h3-h4 or smth and leave the times as p